### PR TITLE
MCKIN-24385 Translations added for 8 languages

### PR DESCRIPTION
**Pull request by musmanmalik**
Translations added for 8 languages

### CODE REVIEW

**Comment by musmanmalik** [comment]
@msaqib52  Please review this PR.

File: diagnostic_feedback/public/js/translations/ar/textjs.js [hunk 1]
```diff
@@ -1,8 +1,4 @@
 
-            (function(global){
```
Comment (wasifarbisoft) [inline diagnostic_feedback/public/js/translations/ar/textjs.js]: Why removing this   "DiagnosticFeedbackXBlockI18N" it is required I guess??

Comment (musmanmalik) [reply diagnostic_feedback/public/js/translations/ar/textjs.js]: I have used the recommended command for compilation where the namespace is not there. I think that was added recently.

Comment (msaqib52) [reply diagnostic_feedback/public/js/translations/ar/textjs.js]: this namespace should not be removed, seems like namespace attribute is missing in compile command in makefile, please add it there and recompile. https://github.com/mckinseyacademy/xblock-diagnosticfeedback/blob/master/Makefile#L36

**Review by msaqib52 — CHANGES_REQUESTED** [review]
@musmanmalik these js translations are not being returned by the student_view, so these translations won't work unless we add them to student view.

**Review by wasifarbisoft — APPROVED** [review]

**Comment by wasifarbisoft** [comment]
@msaqib52 looks good to me! do you see anything :)

**Comment by msaqib52** [comment]
@musmanmalik can u install travis on your fork and run the build to see if all the tests are passing.

**Comment by msaqib52** [comment]
Build is passing https://travis-ci.com/github/msaqib52/xblock-diagnosticfeedback/jobs/429633061

**Pull request merged**